src/palace.pyx

```diff
@@ -2334,6 +2335,8 @@ cdef class ChorusEffect(BaseEffect):
 
     Raises
     ------
+    ValueError
+        If the waveform is neither set to 'sine' nor 'triangle'.
```

Comment:
Not sure if we want this, feeling it's a bit redundant.  @Huy-Ngo, what do you think?

Reply:
Yea I think it's redundant.